common/services/prismic/parsers.js

```diff
@@ -759,3 +762,23 @@ export function parseGenericFields(doc: PrismicFragment): GenericContentFields {
     labels: [],
   };
 }
+
+export function parseItemToFeaturedCard(
```

Comment:
This doesn't feel like a parser, and doesn't really have anything to do with Prismic.

We could keep this in the `FeaturedCard` component as `convertContentToFeatureCardProps` or similar?

Reply:
Yes! This was making me feel weird all along – thanks, will update.